lib/index.js

```diff
@@ -5,10 +5,7 @@ var Resource = require('./Resource'),
     Controllers = require('./Controllers'),
     Errors = require('./Errors'),
     inflection = require('inflection'),
-    _ = require('lodash');
-
-// make deep defaults the default
-_.defaults = require('merge-defaults');
+    mergeDefaults = require('merge-defaults');
```

Comment:
In some cases, Sequelize and Epilogue can share the same lodash. NPM will attempt to reduce amount of packages installed under node_modules directory when packages shared semver-compatible dependencies. I think NPM do that since 2.0.0, not sure...

Example:
```sh
$ npm install epilogue sequelize lodash
$ find node_modules -name lodash
node_modules/epilogue/node_modules/merge-defaults/node_modules/lodash
node_modules/lodash
```
Sequelize and Epilogue shares the same lodash.

--

Why recursive `_.defaults` is bad ? https://github.com/sequelize/sequelize/blob/61713712c8903baa8c54d2f34df04c3ef94a0b33/lib/model.js#L2084-L2091 Sequelize uses _.defaults with `model`, complex data with whole sequelize linked. If a merge recursive is applied to it, we run in ` maximum call stack size exceeded`.

--

Anyway, it is a bad practice to modify an external module behavior.